rootfs/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM  docker.io/library/alpine:3.16
+
+RUN apk add openrc util-linux build-base ca-certificates rng-tools rng-tools-openrc
+RUN rc-update add devfs boot
+RUN rc-update add procfs boot
+RUN rc-update add rngd boot
```

Comment:
Is this the elegant solution to the node-hanging-forever problem?

Reply:
Yes. It isn't new, it's already in the root image, but rngd collects randomness from the cpu random number generator so fills the randomness pool immediately in most cases.